client/src/components/BlockEditor/helpers/convertImportsForNodes.ts

```diff
@@ -0,0 +1,74 @@
+import * as uuidv1 from "uuid/v1";
+
+const DOLLAR_NUMBERS = /\$[\d]*/g;
```

Comment:
same comment re regex above - I wonder if exporting and reusing the same regex might be better than duplication